static/app/stores/hookStore.tsx

```diff
@@ -49,6 +50,9 @@ const storeConfig: Reflux.StoreDefinition & HookStoreInterface = {
  *
  * This functionality is primarily used by the SASS sentry.io product.
  */
-const HookStore = Reflux.createStore(storeConfig) as Reflux.Store & HookStoreInterface;
+
+const HookStore = makeSafeRefluxStore(
+  Reflux.createStore(storeConfig) as HookStoreInterface & SafeRefluxStore
```

Comment:
why did you have to import the `SafeRefluxStore` here?

Reply:
Mistake when I was testing if I can get the proper type inference w/o type casts, then when I couldnt, I cast it to the wrong type. Will fix to Reflux.store